crm_claim_type/README.md

```diff
@@ -0,0 +1,61 @@
+CRM Claim Types
+==============
+
+Includes a way to clasify claims adding the concept of type.
+It gives a section to define a set of claim types.
+
+Installation
+============
+
+To install this module, you just need to select it from availables modules
+
+Configuration
+=============
+
+This modules do not require special configurations.
+It contains a predefined set of claim types.
+But if you want add your own types go to Sales > Configurations > Claims > Claim Types
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to Sales > Configurations > Claims > Claim Types
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/crm/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/crm/issues/new?body=module:%20crm_claim_type%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
It is not correctly formatted because it has **.md** as file extension as you commented [here](https://github.com/OCA/maintainer-tools/pull/94#issuecomment-127211142), but this is recognized when **.rst** is used instead. I renamed it to fix this